docs/_static/documentation.inc

```diff
@@ -3,6 +3,16 @@
 Documentation
 =============
 
-Documentation is available `here <http://nimi-python.readthedocs.io>`_.
+Documentation is available on **Read the Docs**:
+
+- https://nidcpower.readthedocs.io/en/stable
```

Comment:
Is this generated?
Seems like a shame to have to enumerate modules manually.

Reply:
>Is this generated?

Not yet.

Reply:
Maybe in the future we'll change this, but I'm not looking to make bigger decisions like that here.